tools/topology/topology1/sof-rn-rt5682-rt1019.m4

```diff
@@ -21,7 +21,7 @@ include(`platform/amd/acp.m4')
 # Playback pipeline 1 on PCM 0 using max 2 channels of s16le.
 # Schedule 96 frames per 2000us deadline on core 0 with priority 0
 PIPELINE_PCM_ADD(
-ifdef(`DTS', sof/pipe-eq-iir-dts-codec-playback.m4, sof/pipe-passthrough-playback.m4),
+ifdef(`DTS', sof/pipe-dts-codec-eq-iir-playback.m4, sof/pipe-passthrough-playback.m4),
```

Comment:
Will "DTS before EQIIR" be the common case (or even more native in algorithm)?
If yes, we should land on main first.

Reply:
No, this change is for Dewatt to align the CRAS tuning result. I'll still land this commit on main for consistency.